### ISSUE TITLE
Clean up widget titles

### ISSUE THREAD

**Issue by lukefretwell** [proudcity/wp-proudcity#1251]
For the widget titles, looks like we're using h2 (ex: content list) AND h3 (ex: text). Can we default to h2 for all of them?

These are the ones that need updating:

* Auth0 (remove class="widget-title")
* Service center (add title option - currently doesn't have one)
* Text (change to h2 and remove class="widget-title")
* Video (doesn't show title)

## Related

* https://github.com/proudcity/wp-proudcity/issues/1380
* https://github.com/proudcity/wp-proudcity/issues/1409

**Comment by jlyon** [comment]
@aschmoe how hard would this be, do you think?
